packages/db/src/meta/pouch/adapters/fs.ts

```diff
@@ -15,10 +15,8 @@ export interface DatabasesSettings {
   directory: string;
 }
 
-export const getDefaultSettings: GetDefaultSettings = ({
-  workingDirectory
-}) => ({
-  directory: path.join(workingDirectory, ".db", "json")
+export const getDefaultSettings: GetDefaultSettings = ({ directory }) => ({
+  directory: path.join(directory, ".db", "json")
```

Comment:
Going off the wrapper idea, [it's a bit hacky, but] we could just make this not take any options and instead just call the wrapper.

(This is why I was getting at the business about making this something that gets passed to `forDefinitions`)